docs/docs/adrs/adr-005-cryptographic-equivocation-verification.md

```diff
@@ -19,7 +20,7 @@ Every proposal needs to go through a (two weeks) voting period before it can be
 Given a three-week unbonding period, this means that an equivocation proposal needs to be submitted within one week since the infraction occurred.
 
 This ADR proposes a system to slash validators automatically for equivocation, immediately upon the provider chain's receipt of the evidence. Another thing to note is that we intend to introduce this system in stages, since even the partial ability to slash and/or tombstone is a strict improvement in security.
-For the first stage of this work, we will only handle light client attacks.
+The feature will be implemented in two parts, each with its dedicated endpoint. One endpoint will handle light client attacks, while the other will handle double signing attacks.
```

Comment:
Might be better to use present tense here and avoid `will`, etc. as state [here](https://github.com/cosmos/interchain-security/blob/main/docs/docs/adrs/intro.md):
> Note the context/background should be written in the present tense.